TESTING.rst

```diff
@@ -276,3 +276,32 @@ your local sources to the ``/opt/airflow`` location of the sources within the co
 .. image:: images/source_code_mapping_ide.png
     :align: center
     :alt: Source code mapping
+
+
+DAG testing
+===========
+
+To ease and speed up process of developing DAGs you can use ``InProcessExecutor`` - a single
+process executor for debugging purposes. Using this executor you can run and debug DAGs
+from your IDE.
+
+**IDE setup steps:**
+
+1. Add ``main`` block at the end of your DAG file to make it runnable:
+
+  .. code-block:: python
+
+    if __name__ == '__main__':
+      dag.clear(reset_dag_runs=True)
+      dag.run()
```

Comment:
Isn't there already a `dag.cli()` or something?

Reply:
Yes, there is. I would like us to tie in to that command somehow please.

Reply:
`dag.cli()` does not work as far as I know (planned to fix). I was thinking about adding `debug` method, so only this has to be done in DAG, no additional configuration of running a file.